docs/reference/adventureworks/armTemplates/auxiliary/policies.json

```diff
@@ -18097,6 +18097,1275 @@
+                          "value": "[[parameters('vulnerabilityAssessmentsStorageContainerName')]"
+                        }
+                      }
+                    }
+                  }
+                }
+              }
+            }
+          },
+          "name": "Deploy-Synapse-Sql-VulnerabilityAssessment"
+        },
+        {
+          "properties": {
+            "displayName": "Deploy-Synapse-VulnerabilityAssessment",
+            "policyType": "Custom",
+            "mode": "Indexed",
+            "description": "Deploy Synapse workspace vulnerability assessments.",
+            "metadata": {
+              "version": "1.0.0",
+              "category": "Synapse"
+            },
+            "parameters": {
+              "effect": {
+                "type": "String",
+                "metadata": {
+                  "displayName": "Effect",
+                  "description": "Enable or disable the execution of the policy"
+                },
+                "allowedValues": [
+                  "DeployIfNotExists",
+                  "Disabled"
+                ],
+                "defaultValue": "DeployIfNotExists"
+              },
+              "vulnerabilityAssessmentsEmail": {
+                "type": "String",
+                "metadata": {
+                  "description": "The email address to send alerts",
+                  "displayName": "Email Address"
+                }
+              },
+              "vulnerabilityAssessmentsStorageID": {
+                "type": "String",
+                "metadata": {
+                  "description": "The storage account to store assessments",
+                  "displayName": "Storage account"
+                }
+              },
+              "vulnerabilityAssessmentsStorageContainerName": {
+                "type": "String",
+                "metadata": {
+                  "description": "The storage account container name to store assessments",
+                  "displayName": "Storage account container name"
+                }
+              }
+            },
+            "policyRule": {
+              "if": {
+                "allOf": [
+                  {
+                    "field": "type",
+                    "equals": "Microsoft.Synapse/workspaces"
+                  }
+                ]
+              },
+              "then": {
+                "effect": "[[parameters('effect')]",
+                "details": {
+                  "type": "Microsoft.Synapse/workspaces/sqlPools/vulnerabilityAssessments",
+                  "name": "default",
+                  "existenceCondition": {
+                    "field": "Microsoft.Synapse/workspaces/vulnerabilityAssessments/recurringScans.isEnabled",
+                    "equals": "True"
+                  },
+                  "roleDefinitionIds": [
+                    "/providers/Microsoft.Authorization/roleDefinitions/b24988ac-6180-42a0-ab88-20f7382dd24c"
```

Comment:
Is Contributor to high here?

Reply:
There are no Synapse specific RBAC roles available to deploy this.